processor/resourcedetectionprocessor/internal/system/system.go

```diff
@@ -94,6 +107,17 @@ func (d *Detector) Detect(ctx context.Context) (resource pcommon.Resource, schem
 		}
 	}
 
+	var hostMACAttribute []any
```

Comment:
Just a `golang` question out of ignorance, is there a reason to have `any` here instead of `string`?

Reply:
`mdatagen` methods will accept `[]any` when they relate to a slice attribute. Converting from `[]string` to `[]any` means looping through the slice and casting each element, so to avoid doing this I use `[]any` directly.